pkg/sdk/testint/security_integrations_gen_integration_test.go

```diff
@@ -1285,10 +1285,19 @@ func TestInt_SecurityIntegrations(t *testing.T) {
 		err = client.SecurityIntegrations.AlterScim(ctx, unsetRequest)
 		require.NoError(t, err)
 
+		// check setting empty comment because of lacking UNSET COMMENT
```

Comment:
I don't like this logic, as stated in your doc-discuss comment and DOC-2249, if it works for other integrations, it is most probably a SF bug.

Let's leave it like this, but let's create a task to remember about this, so that we can address it when it's fixed. Let's add its number to a TODO comment (and add reference to DOC-2249 in it)

Reply:
Done.